app/assets/stylesheets/card.scss

```diff
@@ -3,5 +3,15 @@
 }
 
 .create-new-card {
-  max-width: 400px;
+  max-width: 300px;
+}
+
+.field_with_errors {
+  padding: 2px;
+  color: red;
```

Comment:
Color `red` should be written in hexadecimal form as `#ff0000`<br>Color literals like `red` should only be used in variable declarations; they should be referred to via variable everywhere else.